files/md.yaml

```diff
@@ -0,0 +1,4 @@
+init_config:
```

Comment:
You don't need this file here since you're defining it in group_vars/mysql-va/vars :)